test/reduce.jl

```diff
@@ -244,6 +244,10 @@ end
     end
 end
 
+# 30462
+# test that there is no out of bounds access
+maximum(randn(128,128))
```

Comment:
I would like to add more/better tests that there are no out of bounds accesses. Is there a recommended way to do this? Just make a function call and rely on `--check-bounds=yes`?

Reply:
You could also test `maximum(fill(-Inf, 128, 128)) === -Inf`.  It's unlikely that uninitialized memory will be a bitpattern that represents `-Inf`.  But we do run all our tests with `--check-bounds=yes` (excepting in the `test/boundscheck*.jl` files)